### DISCUSSION TITLE
Welcome to Validly Discussions!

### DISCUSSION THREAD

**Discussion by Hookyns** [Hookyns/validly#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with **Validly**.

**Comment by Rumeet94** [comment]
Hi! I heard about your solution at the IT conference from @Stepami. I've integrated your library into my project and started using it.

I've written a couple of validators to make it easier to validate passwords and phone numbers, and I'd like to submit a pull request, but I'm getting a 403 error when trying to push. Would it be possible to get the necessary access?

I'd also like to contribute more in the future, if you don't mind.

**Reply by Stepami** [reply]
I guess you can submit PR through fork as I do

**Reply by Hookyns** [reply]
@Rumeet94 
Yeap, exactly. Use the fork and then target the main branch of this (original) repository.

**Reply by Rumeet94** [reply]
Thank you. I've sent the PR.

**Reply by Stepami** [reply]
> I heard about your solution at the IT conference from @Stepami.

it looked like this 😅 
i made a talk about awesome SG libraries and put them together as a microservice template infrastructure to show the overall perfomance benefit

my next step is to finish the template to make it available through the dotnet tooling

<img width="2560" height="1707" alt="image" src="https://github.com/user-attachments/assets/bacd9bde-14ea-47d7-bee5-6739350a27b8" />

[there is also a video from the talk](https://ecode.ozon.tech/talks/b725f3db910e488191e79b9614d7d51f/)

**Reply by Hookyns** [reply]
> > I heard about your solution at the IT conference from @Stepami.
> 
> it looked like this 😅 i made a talk about awesome SG libraries and put them together as a microservice template infrastructure to show the overall perfomance benefit
> 
> my next step is to finish the template to make it available through the dotnet tooling
> 

I used AI to translate the video – at least the part about Validly. 🙂👍

Did I get it right that you're a bit unhappy with the attribute-based approach (Microsoft DataAnnotations-like)? 😆 I'm curious – I'd love to hear what people think about Validly.

BTW: I thought about it before. I think I should be able to create extension (e.g. `Validly.Fluent(ly)`) that would generate a separate fluent validator, similar to how FluentValidation does it.

**Reply by Stepami** [reply]
I am ok with Validly's philosophy 👍🏻
I highlighted the risk of not getting along with the attribute-based approach as the industry is too used to FluentValidator.

It is like suggesting people to watch horizontal videos when they only have Tiktok on their phone
